requirements.txt

```diff
@@ -49,12 +49,6 @@ gevent>=1.3.6
 # Used by semantic parsing code to strip diacritics from unicode strings.
 unidecode
 
-# Used by semantic parsing code to parse SQL
-parsimonious>=0.8.0
-
-# Used by semantic parsing code to format and postprocess SQL
-sqlparse>=0.2.4
-
 # For text normalization
 ftfy
```

Comment:
These text normalisation libraries can also go

Reply:
This one is actually used by the OpenAI word splitter, not the semparse code.